src/IO/InputReader.php

```diff
@@ -61,10 +61,6 @@ public function staticValue(
         $validator = $this->makeValidator($validator, $required);
         $answer = $this->io->askAndValidate($label, $validator, 3, $default);
 
-        if (!is_string($answer) && null !== $answer) {
-            return null;
-        }
-
```

Comment:
We're not actually running into this condition since `$io->askAndValidate` will always return `?string`.